### PR TITLE
feat(README): add example to saveHar conditionally

### PR DESCRIPTION
**Pull request by azuruce**

### CODE REVIEW

**Comment by azuruce** [comment]
Neither saveHar or recordHar will work if not chrome or if interactive.
Alternatively, we probably should make isChrome and isInteractive part of build-in check inside saveHar and recordHar? However, we might also need to make it explicit saveHar was not going to for sure save the Har unless it is chrome and not interactive, by renaming it to "saveHarIfSupported" ...

File: README.md [hunk 1]
```diff
@@ -106,6 +106,27 @@ It allows overriding the value, that is specified in the `cypress.json`
...
+
+afterEach( function () {
+    const { state } = this.currentTest;
+    const isInteractive = Cypress.config('isInteractive');
+    const isChrome = Cypress.browser.name === 'chrome';
+
+    if (!isInteractive && isChrome && state !== 'passed') {
+        cy.saveHar();
+    }
+});
```
Comment (Sija) [inline README.md:128]:
```suggestion
beforeEach(function () {
    const isInteractive = Cypress.config('isInteractive');
    const isChrome = Cypress.browser.name === 'chrome';

    if (!isInteractive && isChrome) {
        cy.recordHar();
    }
});

afterEach(function () {
    const { state } = this.currentTest;
    const isInteractive = Cypress.config('isInteractive');
    const isChrome = Cypress.browser.name === 'chrome';

    if (!isInteractive && isChrome && state !== 'passed') {
        cy.saveHar();
    }
});
```

Comment (azuruce) [reply README.md]: agree to the suggestion. Thanks for pointing out. Fix is just pushed.

File: README.md [hunk 2]
```diff
@@ -106,6 +106,24 @@ It allows overriding the value, that is specified in the `cypress.json`
...
+  }
+});
+afterEach(function () {
+  const { state } = this.currentTest;
+  const isInteractive = Cypress.config('isInteractive');
+  const isChrome = Cypress.browser.name === 'chrome';
+  if (!isInteractive && isChrome && state !== 'passed') {
+    cy.saveHar();
+  }
+});
```
Comment (derevnjuk) [inline README.md:125]: Let's align code with other examples in README.md using the arrow function expression 

```suggestion
beforeEach(() => {
  const isInteractive = Cypress.config('isInteractive');
  const isChrome = Cypress.browser.name === 'chrome';
  if (!isInteractive && isChrome) {
    cy.recordHar();
  }
});
afterEach(() => {
  const { state } = this.currentTest;
  const isInteractive = Cypress.config('isInteractive');
  const isChrome = Cypress.browser.name === 'chrome';
  if (!isInteractive && isChrome && state !== 'passed') {
    cy.saveHar();
  }
});
```

**Review by derevnjuk — APPROVED** [review]

**Comment by derevnjuk** [comment]
> we probably should make isChrome and isInteractive part of build-in check

I guess it's a good point. I agree we have to move such kind of checks to commands. Instead of introducing a new command (such as `saveHarIfSupported`), I suggest improving existing ones. Additionally, we can notify the user if one of the conditions are not met (just raise warning or something else). Wdyt?

File: README.md [hunk 3]
```diff
@@ -106,6 +106,24 @@ It allows overriding the value, that is specified in the `cypress.json`
 cy.saveHar({ outDir: './hars' });
 ```
 
+Generate HAR file only for chrome, if it is not interactive run, and if test failed.
+```
```
Comment (derevnjuk) [inline README.md:110]:
```suggestion
```js
```

**Review by derevnjuk — DISMISSED** [review]

**Comment by derevnjuk** [comment]
I'm gonna merge initial version since there's no activity

**Pull request merged**